### PR TITLE
[Form][SubmitType] Add "validate" option

### PR DESCRIPTION
**Pull request by fancyweb**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | https://github.com/symfony/symfony/issues/8763
| License       | MIT
| Doc PR        | TODO

The second part of the ticket requires more work but is kind of unrelated.

### CODE REVIEW

**Review by darielopper — APPROVED** [review]

**Review by xabbuh — APPROVED** [review]
Having a test for the form theme ensuring that the attribute is actually rendered would be a nice bonus. :)

**Comment by xabbuh** [comment]
@fancyweb Could you rebase here please so that the builds actually pass?

**Comment by fancyweb** [comment]
@xabbuh Done + I added the test.

File: src/Symfony/Component/Form/Tests/AbstractLayoutTest.php [hunk 1]
```diff
@@ -2730,4 +2731,38 @@ public function testButtonWithTranslationParameters()
 '
         );
     }
+
+    /**
+     * @dataProvider submitFormNoValidateProvider
+     */
+    public function testSubmitFormNoValidate(bool $attributeIsExpected, bool $validate)
```
Comment (xabbuh) [inline src/Symfony/Component/Form/Tests/AbstractLayoutTest.php]: we could omit the `$attributeIsExpected` argument here and use `!$validate` instead below

Comment (fancyweb) [reply src/Symfony/Component/Form/Tests/AbstractLayoutTest.php]: Indeed

**Review by yceruto — APPROVED** [review]

**Comment by xabbuh** [comment]
Thank you @fancyweb.

**Pull request merged**